rfcs/0013-synthea-future-claims-and-automation.md

```diff
@@ -0,0 +1,143 @@
+    * [Proposed Solution: Notable Alternatives](#proposed-solution-notable-alternatives)
+* [Prior Art](#prior-art)
+* [Future Possibilities](#future-possibilities)
+* [Addendums](#addendums)
+
+## Motivation
+[Motivation]: #motivation
+
+For those unfamiliar with Mitre's Synthea application for generating synthetic beneficiary and claim data, before discussing motivation and a proposed solution, here is some background on Synthea and how BFD uses it. 
+
+### Background on Synthea
+
+Mitre Synthea:
+
+* [Synthea](https://github.com/synthetichealth/synthea) -- for generating synthetic beneficiary and claim data to load into BFD DB.
+  * How To Differentiate Synthea And Real Data:
+    - Synthetic beneficiary IDs are a negative number.
+    - Synthetic MBIs have the character 'S' in a certain position.
+
+Generating & Loading Synthea Data: 
+- BFD clones the master branch of the Synthea codebase, and generates the RIF files via the `run_Synthea` command, where the batch size and geography can be set, as well as a Synthea.properties file, which is important, for defining numeric ranges of key claim and beneficiary properties i.e. beneficiary ID, claim group ID, claim date, etc.
+- The RIF files are copied into the BFD Server codebase, and loaded into the local database. 
+- If there are no errors loading locally, the RIF files are uploaded to AWS S3 and loaded into TEST, PROD SBX, PROD databses via the same ETL process with CCW RIF files. 
+- Synthetic data is now available to BFD users.
+
+* In addition to these steps, BFD follows a thorough test plan before generating or loading data into TEST, PROD SBX, or PROD environments. 
+  * [Synthea Test Plan](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-model/bfd-model-rif-samples/dev/synthea-test-plan.md) 
+
+Motivation:
+As previous synthetic data releases were separated by 3 months, and only 20,000 synthetic beneficiaries released, data became quickly outdated, and the amount of data being released doesn't justify the usecase of synthetic data. Additionally, there is inefficiency involved with producing and releasing synthetic data when done without automation. Manual process are prone to human error, and can cause incidents, such as accidental overlap of beneficiary IDs between different synthetic data batches. There are more than enough reasons to look into streamlining and scaling up synthetic data production to maximize the benefit to users and partners. This involves automated recurring Synthea data generation, automated recurring loading, large Synthea data generation, and potential required changes to the BFD server and pipeline application.
+
+## Proposed Solution
+[Proposed Solution]: #proposed-solution
+
+Automated Recurring Generation: 
+To increase the benefit and quality of synthetic data to the user, data that is current, or for future use, needs to be generated on a more regular basis. The proposed plan is weekly and quarterly. 
```

Comment:
Has there been any discussion around generating new synthetic claims for existing beneficiaries?  From what I've seen with the first two Synthea releases, the benes don't overlap.  I understand why this is easier/safer, but from the peering APIs perspective, work still needs to be done on their end to handle attribution of the new benes and their synthetic claims in SBX.  If there was a process for adding new claims to existing benes, then it would be easier for peering APIs to make immediate use of this data once it's loaded into SBX.

Reply:
We asked that from Jason Walonoski - if there was a way to generate the bene content as an intermediate state/file that could be reintroduced make into a Synthea run that only generated new claims and not additional benes. I think they are interested in this concept, but it's not currently possible based on object serialization issues.

Reply:
There are two solutions here..

One is generating claims into the future, and loading those previously "future claims" on a weekly basis so the beneficiaries appear to be updating. Our teams are collectively working on that now.

The second solution is to regenerate the same beneficiaries with new claims. That is harder, and will take longer, but we're working on it.

The first solution works, but the second is preferable, because you'd take advantage of all the updates and fixes in the second scenario.